runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -417,6 +417,11 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 
+		if (chatMessage.equals("You have been doomed!") && config.screenshotPlayerDeath())
```

Comment:
Isn't this message colored ingame? If so, this would not match. Was this tested?